helpers/mathjax.js

```diff
@@ -19,6 +19,7 @@ export class HtmlBlockMathRenderer {
 		if (!elem.querySelector('math') && !(isLatexSupported && /\$\$|\\\(|\\\[|\\begin{|\\ref{|\\eqref{/.test(elem.innerHTML))) return elem;
 
 		const mathJaxConfig = {
+			deferTypeset: true,
```

Comment:
This needs to be a config because we'll for sure want to do initial typeset when we load MathJax into content documents or the equation editor's dialog.